docs/rules/arrow-parens/index.html

```diff
@@ -4,15 +4,15 @@
 rationale: Maintains stylistic consistency with other arrow function definitions.
 optionsDescription: |-
 
-  if `ban-single-arg-parens` is specified, then arrow functions with one parameter 
+  if `ban-single-arg-parens` is specified, then arrow functions with one parameter
   must not have parentheses if removing them is allowed by TypeScript.
 options:
   type: string
   enum:
     - ban-single-arg-parens
 optionExamples:
   - 'true'
-  - '[true, "ban-single-arg-parens"]'
+  - '[true, "ban-single-arg-parens]"'
```

Comment:
`docs` is generated, so this is due to an unrelated change (#1947). Made fix in #1967 and filed  #1968.